probe-rs-tools/src/bin/probe-rs/util/rtt/client.rs

```diff
@@ -49,11 +49,26 @@ impl RttClient {
         };
 
         if let Some(elf) = elf {
-            if let Some(address) = RttActiveTarget::get_rtt_symbol_from_bytes(elf) {
-                this.scan_region = ScanRegion::Exact(address);
-                this.core_id = target.core_index_by_address(address).unwrap_or(0);
+            let mut init_defmt = false;
+            match RttActiveTarget::get_rtt_symbol_from_bytes(elf) {
```

Comment:
This is a second line of defense that fixes the debugger - I'm too lazy to find format options there, and I guess if we've loaded a firmware we might want to verify that we can work with it.